pkg/templates/notificationtemplates/notification_generator.go

```diff
@@ -1 +1,61 @@
 package notificationtemplates
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+var log = logf.Log.WithName("notification-templates")
+
+// NotificationTemplate contains the template subject and content
+type NotificationTemplate struct {
+	Subject string
+	Content string
+}
+
+// GetNotificationTemplates returns a notification subject and body or an error
+func GetNotificationTemplates(name string, assets Assets) (*NotificationTemplate, error) {
```

Comment:
we could skip the `assets` parameter here, so it's easier to use. But if we need to pass some "test" assets, we could use a make this func variadic and have an option such as "WithAssets(...)".

Reply:
done as part of: 32f0c75bea7b78ec9fd48726758ade2ba1b1ab66